src/components/ha-selector/ha-selector-number.ts

```diff
@@ -81,17 +81,11 @@ export class HaNumberSelector extends LitElement {
     `;
   }
 
-  private get _value() {
-    return this.value ?? (this.selector.number?.min || 0);
-  }
-
   private _handleInputChange(ev) {
     ev.stopPropagation();
     const value =
       ev.target.value === "" || isNaN(ev.target.value)
-        ? this.required
-          ? this.selector.number?.min || 0
-          : undefined
+        ? ""
```

Comment:
Why change this to an empty string instead of `undefined`?

Reply:
@piitaya

Reply:
Changed